tests/yam/agama/validate_headless.pm

```diff
@@ -0,0 +1,32 @@
+## Copyright 2025 SUSE LLC
```

Comment:
In the future the content of this test module should go after boot_agama, but we would need some sort of library (which I am really afraid to create for now due to code maintenance) due it needs to be reused for qemu and other remote backends. This part ensures how the system boot, so it is part of the boot, but for now it is fine in a separate module, could you please file a brief  progress ticket with this to not forget.

The final goal is to have less schedules, so most of the test suite would use similar path with some adjustments.

Reply:
Filed [ticket#181607](https://progress.opensuse.org/issues/181607). and please take a look againa for this PR. Thanks